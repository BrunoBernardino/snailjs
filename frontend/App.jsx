// @flow
import React from 'react';

import Router from 'Router';

import 'App.scss';

const App = () => (
  <div className="App">
    <Router />
  </div>
);

export default App;
