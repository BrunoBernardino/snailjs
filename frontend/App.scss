html,
body {
  margin: 0;
  padding: 0;
  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
}

.App {
  padding: 20px;
  display: flex;
  justify-content: center;
  align-items: center;
  text-align: center;

  img {
    text-align: center;
    width: 100%;
    max-width: 350px;
  }
}
