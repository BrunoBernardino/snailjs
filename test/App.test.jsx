import React from 'react';
import renderer from 'react-test-renderer';
import App from 'App';

test('App mounts without failing', () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
