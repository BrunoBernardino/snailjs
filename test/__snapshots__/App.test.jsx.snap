// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App mounts without failing 1`] = `
<div
  className="App"
>
  <div
    className="Counter"
  >
    <h1
      className="Counter__Title-sc-15vxb1x-0 frBzxp"
    >
      Welcome to SnailJS!
    </h1>
    <img
      alt="Logo"
      src="/static/logo.png"
    />
    <button
      className="Counter__StyledButton-sc-15vxb1x-1 ixzgSS Button__StyledButton-sc-1at07yy-0 cVUxdz"
      onClick={[Function]}
      type="button"
    >
      This button has been clicked 
      0
       time
      s
      !
    </button>
    <section
      className="Counter__LinkWrapper-sc-15vxb1x-2 isVwwO"
    >
      <a
        className="Link__StyledLink-sc-1ukbgas-0 huLbfE"
        href="/6"
        onClick={[Function]}
      >
        Start with 6 clicks
      </a>
      
      <a
        className="Link__StyledAnchor-sc-1ukbgas-1 cZfESO"
        href="/server?apiKey="
      >
        Server-only route
      </a>
    </section>
  </div>
</div>
`;
